packages/amplify_auth_cognito/lib/amplify_auth_cognito_stream_controller.dart

```diff
@@ -17,10 +17,14 @@ import 'dart:async';
 import 'package:flutter/services.dart';
 import 'src/CognitoHubEvents/AuthHubEvent.dart';
 
+/// An event channel for receiving HUB events over a PlatformChannel.
 EventChannel channel = const EventChannel("com.amazonaws.amplify/auth_cognito_events");
+/// A subscription established for the event stream.
 StreamSubscription eventStream;
 
+/// Provides a StreamController for HUB
 class AuthStreamController {
+  /// The getter for the _authStreamController
```

Comment:
The AuthStreamController class is used externally by amplify_flutter, so I don't believe it can be private - but I have noted in the docs that it is 'for internal usage'.  The channel and eventStream I've marked as private.